SurveyApp/Modules/Home/HomeViewController.swift

```diff
@@ -35,14 +37,7 @@ final class HomeViewController: UIViewController {
         collectionViewLayout: AnimatedCollectionViewLayout(animationStyle: .fade)
     )
     
-    // TODO: replace hard code use case with list of Survey objects from API
-    private let backgrounds = [Asset.sampleBackground3(), Asset.sampleBackground2(), Asset.sampleBackground1()]
-    private let titles = ["Working from home Check-In", "Career training and development", "Inclusion and belonging"]
-    private let descriptions = [
-        "We would like to know how you feel about our work from home...",
-        "We would like to know what are your goals and skills you wanted...",
-        "Building a workplace culture that prioritizes belonging and inclusio..."
-    ]
+    private var surveys: [Survey] = []
```

Comment:
I think this should be ViewModel for the cell to keep View from knowing underlining entity. Maybe create `SurveyCellViewModel` and output from presenter.

Reply:
I updated the code. Please help review again. 🙏